x/subscription/migrations/v4/params.go

```diff
@@ -0,0 +1,39 @@
+package types
+
+import (
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"gopkg.in/yaml.v2"
+)
+
```

Comment:
why do we need params?

Reply:
query.pb.go uses params.pb.go (in the GetParams query)
params.pb.go uses params.go because the String() method is implemented in params.go